### PR TITLE
Fix user lookups for numeric usernames II: PHP Strikes Back

### PR DESCRIPTION
**Pull request by cl8n**

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -362,7 +362,7 @@ public static function lookup($username_or_id, $lookup_type = null, $find_all =
                 break;
 
             default:
-                if (is_numeric($username_or_id)) {
+                if (ctype_digit($username_or_id)) {
```
Comment (nanaya) [inline app/Models/User.php]: `ctype_digit(1)` is false

Comment (cl8n) [reply app/Models/User.php]: it's a string though

Comment (nanaya) [reply app/Models/User.php]: but is it (it's called in quite a few places)

Comment (cl8n) [reply app/Models/User.php]: yeah, every call (with default lookup) is a string. should I cast it just to be safe?

Comment (nanaya) [reply app/Models/User.php]: yes, before someone gets confused why `lookup(48)` works but returns wrong user.

**Pull request merged**